src/main/java/com/fasterxml/jackson/databind/deser/impl/JDKValueInstantiators.java

```diff
@@ -44,53 +54,129 @@ public static ValueInstantiator findStdValueInstantiator(DeserializationConfig c
             if (raw == HashMap.class) {
                 return HashMapInstantiator.INSTANCE;
             }
+            if (raw == ConcurrentHashMap.class) {
+                return ConcurrentHashMapInstantiator.INSTANCE;
+            }
+            if (raw == TreeMap.class) {
+                return TreeMapInstantiator.INSTANCE;
+            }
             if (Collections.EMPTY_MAP.getClass() == raw) {
-                return new ConstantValueInstantiator(Collections.EMPTY_MAP);
+                return ConstantValueInstantiator.EMPTY_MAP;
             }
         }
         return null;
     }
 
-    private static class ArrayListInstantiator
+    private abstract static class JDKValueInstantiator
         extends ValueInstantiator.Base
         implements java.io.Serializable
+    {
+        public JDKValueInstantiator(Class<?> type) {
+            super(type);
+        }
+
+        @Override
+        public boolean canInstantiate() { return true; }
+
+        @Override
+        public boolean canCreateUsingDefault() {  return true; }
+    }
+
+    private static class ArrayListInstantiator
+        extends JDKValueInstantiator
     {
         private static final long serialVersionUID = 2L;
 
-        public final static ArrayListInstantiator INSTANCE = new ArrayListInstantiator();
+        public static final ArrayListInstantiator INSTANCE = new ArrayListInstantiator();
+
         public ArrayListInstantiator() {
             super(ArrayList.class);
         }
 
         @Override
-        public boolean canInstantiate() { return true; }
+        public Object createUsingDefault(DeserializationContext ctxt) throws IOException {
+            return new ArrayList<>();
+        }
+    }
+
+    private static class LinkedListInstantiator
```

Comment:
Can we add this comment (or similar) for all newly added classes? Or JavaDoc maybe. 
Just so we know 🙂 when we added these.

```suggestion
    // @since 2.17 [databind#4299] Instantiators for additional container classes
    private static class LinkedListInstantiator
```